docs/BasicCommunication/OnSystemRequest/index.md

```diff
@@ -56,6 +60,11 @@ BC.OnSystemRequest in "Proprietary" Policy Table Update Flow
 ![Proprietary PTU](./assets/OnSystemRequest_in_Proprietary_PTU_flow.png)
 |||
 
+|||
+_EXTERNAL proprietary_ Policy Table Update Flow
+![EXTERNAL proprietary](./assets/diagram_PolicyUpdate_external_proprietary.png) 
```

Comment:
This link is broken because the diagram is in the `PolicyUpdate` assets folder for some reason.